sonar-ballerina-plugin/src/main/resources/io/ballerina/sonar/ballerina-1.html

```diff
@@ -0,0 +1,48 @@
+<p>
+    When 'checkpanic' is used, the program terminates abruptly with a panic unless it’s handled explicitly along the call stack.
+</p>
+
+<h2>Noncompliant Code Example</h2>
+
+<pre>
+public function checkResult() {
+    json result = checkpanic getResult();
```

Comment:
This example contains a unused variable warning
Shall we do `json result = checkpanic getResult();` or add a print statment

Reply:
After an offline discussion with @SasinduDilshara resorted to keep the result variable.